src/main.rs

```diff
@@ -96,7 +89,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
     println!("Deleted {} documents", delete_result.deleted_count);
 
     // Working with Document is a bit horrible:
-    if let Some(title) = new_doc.get("title").and_then(Bson::as_str) {
+    if let Ok(title) = new_doc.get_str("title") {
```

Comment:
Hopefully this is a bit less horrible :)

Reply:
That is a lot less horrible :)

Reply:
... although the real point of that line was to show the advantages of using serde.